php/tests/vaas/VaasTest.php

```diff
@@ -557,29 +623,59 @@ public function testForStream_WithCleanUrlContentAsStream_ReturnsClean()
      */
     public function testForStream_WithEicarUrlContentAsStream_ReturnsMalicious()
     {
+        $request = new Request(self::MALICIOUS_URL);
+        try {
+            $response = $this->httpClient->request($request);
+            if ($response->getStatus() != 200) {
+                throw new Exception($response->getReason(), $response->getStatus());
+            }
+        } catch(Exception $e) {
+            echo $e->getMessage();
+        }
+        $body = $response->getBody();
+        $size = $response->getHeader("Content-Length");
+
         $vaas = $this->_getVaas();
         $vaas->Connect($this->getClientCredentialsGrantAuthenticator()->getToken());
-        $httpClient = new Client();
-        $response = $httpClient->get(self::MALICIOUS_URL);
-        $stream = new Stream($response->getBody()->detach());
-
-        $verdict = $vaas->ForStream($stream);
+        $verdict = $vaas->ForStream($body, $size);
 
         $this->assertEquals(Verdict::MALICIOUS, $verdict->Verdict);
     }
 
     public function testForStream_WithEicarUrlContentAsStream_ReturnsMaliciousWithDetectionAndMimeType()
     {
+        $request = new Request(self::MALICIOUS_URL);
+        try {
+            $response = $this->httpClient->request($request);
+            if ($response->getStatus() != 200) {
+                throw new Exception($response->getReason(), $response->getStatus());
+            }
+        } catch(Exception $e) {
+            echo $e->getMessage();
+        }
+        $body = $response->getBody();
+        $size = $response->getHeader("Content-Length");
+
         $vaas = $this->_getVaas();
         $vaas->Connect($this->getClientCredentialsGrantAuthenticator()->getToken());
-        $httpClient = new Client();
-        $response = $httpClient->get(self::MALICIOUS_URL);
-        $stream = new Stream($response->getBody()->detach());
-
-        $verdict = $vaas->ForStream($stream);
+        $verdict = $vaas->ForStream($body, $size);
 
         $this->assertEquals(Verdict::MALICIOUS, $verdict->Verdict);
         $this->assertEquals("text/plain", $verdict->MimeType);
         $this->assertNotEmpty($verdict->Detection);
     }
+
+    static function random_strings($length_of_string) {
```

Comment:
There is a duplicate function here. https://github.com/GDATASoftwareAG/vaas/blob/48121e033dae91af41fb3b3b43e122175133b97a/php/tests/vaas/StreamsInLoopTest.php#L29

Reply:
Ah yes. I had problems calling stuff from another class and just copied it here. And in the end, I don't need it anymore. Will remove it.